README.md

```diff
@@ -1 +1,70 @@
-# To-Do-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do-List
+
+In this project, We build a to do list minimalist websites that allows users to create and modify a to do list.
+
+## Built With
+
+* HTML
+* CSS
+* Javascript 
+* Webpack
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+* Get the directory that you want to clone the repository.
+* Open the command prompt in this directory.
+* Clone by using this command : ```git clone https://github.com/Daron976/Awesome-Books.git```
+* Got to the cloned folder in your prompt command using ```cd awesome-books```
+* Run the code.
+* Run `npm start` in order to start a local live server.
+
+### Prerequisites
+
+HTML CSS JavaScript Webpack
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] To keep readme professional, please add section body under headers or delete them.